zeebe-cluster/values.yaml

```diff
@@ -13,6 +13,7 @@ partitionCount: "3"
 replicationFactor: "3"
 cpuThreadCount: "2"
 ioThreadCount: "2"
+zeebeCfg: null
```

Comment:
@Zelldon I don't think that we need this line. I think that we can add some checks in the `configmap.yaml` file to make sure that if .Values.zeebeCfg is not present anything breaks... 
Let's try to keep the values files as simple as possible.

Reply:
Yep we can remove it was not 100% sure if we need to declare this here in order to use it :man_shrugging: sorry I'm total beginner :D

Reply:
@Zelldon no worries at all.. I would appreciate a PR to `github.com/zeebe-io/zeebe-helm`.. with some dev instructions on how to test local changes to the chart.. something that I am running here to check issues is `helm lint` inside the chart directory to at least make sure that the chart is not broken.

Reply:
I used `helm install . --dry-run --debug` as you mentioned to me